src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -2433,26 +2433,22 @@ Replication State Storage::
 `ReplicationPeerStorage` and `ReplicationQueueStorage`. The former one is for storing the
 replication peer related states, and the latter one is for storing the replication queue related
 states.
-  HBASE-15867 is only half done, as although we have abstract these two interfaces, we still only
-have zookeeper based implementations.
+  And in HBASE-27109, we have implemented the `ReplicationQueueStorage` interface to store the replication queue in the hbase:replication table.
 
 Replication State in ZooKeeper::
   By default, the state is contained in the base node _/hbase/replication_.
-  Usually this nodes contains two child nodes, the `peers` znode is for storing replication peer
-state, and the `rs` znodes is for storing replication queue state.
+  After 3.0.0, it only contains one child node, but before 3.0.0, we still use zk to store queue data.
 
 The `Peers` Znode::
   The `peers` znode is stored in _/hbase/replication/peers_ by default.
   It consists of a list of all peer replication clusters, along with the status of each of them.
   The value of each peer is its cluster key, which is provided in the HBase Shell.
   The cluster key contains a list of ZooKeeper nodes in the cluster's quorum, the client port for the ZooKeeper quorum, and the base znode for HBase in HDFS on that cluster.
 
-The `RS` Znode::
```

Comment:
We'd better keep this unchanged, as it describes what we have before 3.0.0. And we can introduce a new section to describe the hbase:replication table storage.